worlds/meritous/Regions.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2022 FelicitusNeko
+#
+# This software is released under the MIT License.
+# https://opensource.org/licenses/MIT
+
+import typing
+from BaseClasses import MultiWorld, Region, Entrance, Location, RegionType
+from .Locations import MeritousLocation, alpha_store, beta_store, gamma_store, chest_store, special_store, location_table
+
+meritous_regions = ["Atlas Dome", "Chest rewards",
+                    "PSI Keys", "Meridian", "Ataraxia", "Merodach", "Endgame"]
```

Comment:
These regions seem like overkill. It seems better to me to just define rules on each location corresponding to the requirements needed, since the three boss locations each only need one item and contain only one location each, it seems. Since the entrances for `Chest rewards` and `PSI Keys` don't seem to have requirements either, your regions are basically not doing anything.

Reply:
Each region seems to have 24 locations in them, 24 locations with the same rule, seems to make sence to me to use a region for that